app.js

```diff
@@ -1,3 +1,4 @@
+//Nick's code
```

Comment:
We have `git blame` that tells us who writes a certain line of code.
I think it would be better to remove comments like this to avoid clutter.